data/json/monsters/mutant_mammal.json

```diff
@@ -104,37 +104,10 @@
     "type": "MONSTER",
     "name": { "str": "iridescent cat" },
     "description": "As it moves, this cat's colors shift unnaturally in the light.  You can't make out the breed, but its fur pattern seems to change with every step.",
-    "default_faction": "cat",
-    "bodytype": "dog",
-    "categories": [ "WILDLIFE" ],
-    "species": [ "MAMMAL" ],
-    "volume": "4000 ml",
-    "weight": "4 kg",
-    "hp": 12,
-    "speed": 150,
-    "material": [ "flesh" ],
-    "symbol": "c",
-    "color": "light_gray",
-    "luminance": 4,
-    "looks_like": "mon_cat",
-    "aggression": -99,
-    "morale": 2,
-    "melee_skill": 6,
-    "melee_dice": 1,
-    "melee_dice_sides": 2,
-    "melee_damage": [ { "damage_type": "cut", "amount": 2 } ],
-    "dodge": 8,
-    "vision_night": 20,
-    "special_attacks": [ { "id": "scratch", "attack_upper": false, "cooldown": 10 } ],
-    "fear_triggers": [ "SOUND", "PLAYER_CLOSE", "FRIEND_ATTACKED" ],
-    "placate_triggers": [ "MEAT" ],
-    "harvest": "mammal_tiny",
+    "copy-from": "mon_cat",
+    "//": "Higher dodge due to the constantly changing fur pattern",
+    "dodge": 9,
```

Comment:
I don't see how the comment justifies the buff, its flesh doesn't change shape because of color swaps

Reply:
My reasoning is because its appearance changes, it is harder to target since it will "confuse" most attackers. This is not a high bonus, "just" +1 dodge.
Its fur is also somewhat luminant, making this relevant.